packages/terra-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -197,20 +197,20 @@ function Cell(props) {
   if (isMasked) {
     cellContent = (
       <span className={cx('no-data-cell', theme.className)}>
-        {intl.formatMessage({ id: 'Terra.worklistDataGrid.maskedCell' })}
+        {intl.formatMessage({ id: 'Terra.data-grid.maskedCell' })}
       </span>
     );
   } else if (!children) {
     cellContent = (
       <span className={cx('no-data-cell', theme.className)}>
-        {intl.formatMessage({ id: 'Terra.worklistDataGrid.blank' })}
+        {intl.formatMessage({ id: 'Terra.data-grid.blank' })}
       </span>
     );
   } else {
     cellContent = children;
   }
 
-  const className = cx('worklist-data-grid-cell', {
+  const className = cx('data-grid-cell', {
```

Comment:
I would just use a class name of cell instead of data-grid-cell.  This component will be part of the Terra Table component.  This will just lead to the need for an additional refactor.

Reply:
Updated